src/plays/color-palettes-generator/ColorPalettesGenerator.js

```diff
@@ -0,0 +1,54 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import React, { useState } from 'react';
+import SingleColor from './components/SingleColor';
+import Values from 'values.js';
+
+// WARNING: Do not change the entry componenet name
+function ColorPalettesGenerator(props) {
+  // Your Code Start below.
+  const [color, setColor] = useState('');
+  const [error, setError] = useState(false);
+  const [list, setList] = useState(new Values('#f15025').all(10));
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    try {
+      let colors = new Values(color).all(10);
+      setList(colors);
+      setError(false);
+    } catch (error) {
+      setError(true);
+    }
+  };
+
+  return (
+    <>

```

Comment:
do we need this fragment? if not then can we remove it?

Reply:
Yeah Sure! Done 👍